public/chat_header_button.tsx

```diff
@@ -119,12 +122,36 @@ export const HeaderChatButton = (props: HeaderChatButtonProps) => {
     }
   };
 
+  useEffect(() => {
```

Comment:
Shall we move `useEffect` to the bottom of `useCallback`?

Reply:
Do you mean `setMountPoint`? I dont think declaration order will have any difference because `SetMountPoint` will always excute after `sidecar().open()`.